### ISSUE TITLE
[Question] Rest API works only with tenant credentials ?

### ISSUE THREAD

**Issue by sebsoft** [thingsboard/thingsboard#5321]
**Component**
 * Thingsboard rest API

**Description**
It looks like you need credentials at tenant level to use the rest API.  We would like to give customers acces to their device data using the REST api but this seems not possible. Is this correct ?

**Comment by ashvayka** [comment]
No, this is not correct, Customers may also query devices and their data. Otherwise, the dashboards you assign to customers would not be able to work.
